config/sidebar.yml

```diff
@@ -223,6 +223,9 @@ articles:
       - title: "Verify Access Tokens"
         url: "/api-auth/tutorials/verify-access-token"
 
+      - title: "Default Login Pages"
```

Comment:
I'm not sure of the placement in the sidebar - why here? Would it better belong in the hosted-pages section, or the management api one?

Reply:
Ok, moved it to hosted-pages